test/integration/send_slide_to_sign_test.rb

```diff
@@ -16,9 +16,10 @@ class SendSlideToSignTest < Capybara::Rails::TestCase
     assert_not slide_is_on_sign?('James Slide', signs(:one))
     sign_out
     sign_in users(:ross)
+
     assert page.has_content?(/James Slide needs to be approved to .* Ross Sign/)
-    assert page.has_content?(/October 5, 2016/)
-    assert page.has_no_content?(/Ending on/)
+    assert page.has_content?(/Starting on/) #slide has a start-date which should appear
+    assert page.has_no_content?(/Ending on/) #don't show end-date if slide doesn't have one
```

Comment:
I've never heard that ruby discourages inline comments in all situations. If this is something we have decided we should put it in our style guide.